pkgs/by-name/hw/hwinfo/package.nix

```diff
@@ -63,6 +59,17 @@ stdenv.mkDerivation (finalAttrs: {
 
   passthru.tests = {
     version = testers.testVersion { package = finalAttrs.finalPackage; };
+    no-usr = testers.testEqualContents {
+      assertion = "There should be no /usr/ paths in the binaries";
+      # There is a bash script that refers to lshal, which is deprecated and not available in Nixpkgs.
+      # We'll allow this line, but nothing else.
+      expected = writeText "expected" ''
+        if [ -x /usr/bin/lshal ] ; then
```

Comment:
How does this work? It looks like an unfinished if-block?

Reply:
Ah, I understand now. The contents of the "expected" derivation output is compared to the contents of "actual".